std-bits/table/src/main/java/org/enso/table/data/column/operation/StorageIterators.java

```diff
@@ -177,6 +184,25 @@ public static <B extends BuilderForType<T>, T> ColumnStorage<T> buildOverLongSto
     return builder.seal();
   }
 
+  private static <B extends BuilderForType<T>, T> ColumnStorage<T> buildOverLongArrayStorage(
+      ColumnLongStorageWithArray source,
+      boolean preserveNothing,
+      B builder,
+      LongBuildOperation<B> operation) {
+    var data = source.getArray();
+    Context context = Context.getCurrent();
+    for (int i = 0; i < source.getSize(); i++) {
```

Comment:
I know we know that `getSize` fits in int but technically it is a `long`, shall we add
```
assert source.getSize() <= Integer.MAX_VALUE;
```
?

Reply:
As this was a specialised path, I didn't want to add anything extra, but there is no harm in adding that for clarity.

Reply:
I think it should be safe - the assertion is run only once and the hot path is the inside of the loop that is run through the many iterations.